tests/rados/test_mgr_daemon.py

```diff
@@ -0,0 +1,134 @@
+"""
+Module to test MGR functionalities
+
+"""
+
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.mgr_workflows import MgrWorkflows
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH manager daemon functional testing
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
```

Comment:
Please collect info on all the mgr daemons present in the cluster before beginning all the tests scenarios. 

Since the original issue was that the mgr daemon was masked lost, and removed from ceph status, we need to check if all the mgr daemons in the beginning of the test are also present post every scenario executed.